packages/p2p-v2/src/providers/AdvertiserPaymentMethodsProvider/advertiserPaymentMethodsReducer.tsx

```diff
@@ -0,0 +1,60 @@
+import { TAdvertiserPaymentMethodsConfig, TReducerAction } from '../types';
+
+const advertiserPaymentMethodsReducer = (
```

Comment:
```suggestion
const paymentMethodsReducer = (
```
Maybe this naming is shorter? `paymentMethodsReducer`

Reply:
Alright @adrienne-deriv, but bare in mind that these are advertiser payment methods (payment methods associated to the advertiser), which is why I prepended `advertiser` to the name